frontend/components/account/Profile.tsx

```diff
@@ -0,0 +1,87 @@
+import { useState } from 'react'
+import { InputField, OptionsField } from '../ui/custom-input'
+import { toast } from 'sonner'
+
+interface IProfileFormInput {
+    username: string
+    proficiency: string
+}
+
+function Profile() {
+    const initialValues: IProfileFormInput = {
+        username: '',
```

Comment:
just a reminder that this should be pre-populated with the user's current username, so maybe can leave a note or do some mock data :)

Reply:
I will not be putting the placeholder username for now, as we still need to link the backend service by this weekend, and it will be used to show the current username instead.

Reply:
lgtm